ERCS/erc-dapp-security.md

```diff
@@ -0,0 +1,111 @@
+---
+title: dApp Security Policy Standard
+description: Defines a document standard for defining the expected on-chain behavior of a dApp frontend.
+author: Bernard Wagner (bernard-wagner)
+status: Draft
+discussions-to: https://ethereum-magicians.org/t/dapp-security-policy/21431
```

Comment:
The discussions link should be in the ERC category on Eth Magicians.  You could move it and add the ERC number or create a new discussions topic which is just a link to this PR.